core/internals.h

```diff
@@ -74,8 +76,11 @@
 #define URI_REGISTRATION_SEGMENT_LEN    2
 #define URI_BOOTSTRAP_SEGMENT           "bs"
 #define URI_BOOTSTRAP_SEGMENT_LEN       2
+#define URI_DELETE_ALL_SEGMENT          "/"
+#define URI_DELETE_ALL_SEGMENT_LEN      1
```

Comment:
Are you sure this is working ?
Because in CoAP the "/" URI is expressed as having no Uri-Path option in the message.